### PR TITLE
perlPackages.CacheKyotoTycoon: init at 0.16

### PR DESCRIPTION
**Pull request by tekeri**
###### Motivation for this change
Add a missing perl package

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @puckipedia @stigtsp @7c6f434c

### CODE REVIEW

**Comment by tekeri** [comment]
deps https://github.com/NixOS/nixpkgs/pull/73201

**Comment by stigtsp** [comment]
> deps #73201

I'd include the dependency in this PR.

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -1175,6 +1175,21 @@ let
     };
   };
 
+  CacheKyotoTycoon = buildPerlModule {
+    pname = "Cache-KyotoTycoon";
+    version = "0.16";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/T/TO/TOKUHIROM/Cache-KyotoTycoon-0.16.tar.gz;
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix]: Please quote the URL. https://github.com/NixOS/rfcs/pull/45

**Comment by tekeri** [comment]
squashed

**Comment by stigtsp** [comment]
The PR name, and commit message(s), do not match the derivation names. For example: `perlPackages.Cache-KyotoTycoon: init at 0.16` should be `perlPackages.CacheKyotoTycoon: init at 0.16`, and `perlPackages-Furl-3.13` should be `perlPackages.Furl: init at 3.13`.

**Comment by stigtsp** [comment]
Tested Furl with:
```
nix-shell -p 'perl.withPackages(p: [p.Furl p.IOSocketSSL])' --run "perl -MFurl -e 'print Furl->new->get(qq(https://nixos.org/))->status_line'"
200 OK⏎
```

**Comment by stigtsp** [comment]
Tested CacheKyotoTycoon by loading it only:

```
nix-shell -p 'perl.withPackages(p: [p.CacheKyotoTycoon])' --run "perl -MCache::KyotoTycoon -le 'print 1'"
1
```

**Review by stigtsp — APPROVED** [review]
Looks good to me

**Comment by tekeri** [comment]
Thank you for review and merge @stigtsp, @7c6f434c

**Pull request merged**